jte/src/main/java/gg/jte/compiler/IoUtils.java

```diff
@@ -18,17 +21,14 @@ public static String toString(InputStream inputStream) throws IOException {
     }
 
     public static void deleteDirectoryContent(Path directory) {
-        if (!Files.exists(directory)) {
```

Comment:
I think we should keep this check. If there's no directory to delete, we don't need to try stuff and raise an exception.

Reply:
Technically it won't since I'm catching the exception and ignoring it. I did this to try to lower the cyclomatic complexity but, as you can see, it didn't really work - happy to revert back.